backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -64,4 +69,13 @@ public void updateUserById(
 		userService.updateUser(user.getId(), userUpdateModel.getEmail());
 	}	
 
+	@PostMapping
+	@Operation(summary = "Create a new user ")
+	public void createUserByEmailAndUserAttributes(
```

Comment:
This method should have a `@ResponseStatus(HttpStatus.NO_CONTENT)` to indicate that it returns a 204 status code.